pkg/cmd/issue/edit/edit.go

```diff
@@ -96,12 +95,15 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.Editable.Title, "title", "t", "", "Revise the issue title.")
-	cmd.Flags().StringVarP(&opts.Editable.Body, "body", "b", "", "Revise the issue body.")
-	cmd.Flags().StringSliceVarP(&opts.Editable.Assignees, "assignee", "a", nil, "Set assigned people by their `login`. Use \"@me\" to self-assign.")
-	cmd.Flags().StringSliceVarP(&opts.Editable.Labels, "label", "l", nil, "Set the issue labels by `name`")
-	cmd.Flags().StringSliceVarP(&opts.Editable.Projects, "project", "p", nil, "Set the projects the issue belongs to by `name`")
-	cmd.Flags().StringVarP(&opts.Editable.Milestone, "milestone", "m", "", "Set the milestone the issue belongs to by `name`")
+	cmd.Flags().StringVarP(&opts.Editable.Title.Value, "title", "t", "", "Edit the title.")
+	cmd.Flags().StringVarP(&opts.Editable.Body.Value, "body", "b", "", "Edit the body.")
```

Comment:
To be more clear, can we describe these flags as "Set the new title/body", since that is what they do?